test_unstructured_inference/test_api.py

```diff
@@ -53,89 +50,10 @@ def test_layout_parsing_api(monkeypatch, filetype, ext, data, process_func, expe
 def test_bad_route_404():
     client = TestClient(api.app)
     filename = os.path.join("sample-docs", "loremipsum.pdf")
-    response = client.post("/layout/badroute", files={"file": (filename, open(filename, "rb"))})
-    assert response.status_code == 404
-
-
-def test_layout_v02_api_parsing_image():
```

Comment:
Looks like a lot of these tests were moved to `test_yolox.py`. Did you make changes to these tests or did they just move?

Reply:
Just moved to that file